integration_test/build-adapters.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+base_name="gcr.io/${1}/healthcare-api-dicom-dicomweb-adapter-"
+
+build_adapter () {
+  adapter_name=${base_name}${1}
+  version=$(cat build.gradle | grep "version = '"| grep -Eo "[0-9]{1,3}\.[0-9]{1,3}")
+  postfix=$(gcloud container images list-tags "${adapter_name}" --sort-by=~tags | grep " $version" |  grep -Eo "([0-9]{1,3}\.){2}[0-9]{1,3}" | head -1 | grep -Eo "[0-9]{1,3}" | tail -1)
```

Comment:
I think the default order of ~timestamp would be better than by tag. Incrementing past 9 changes the last number to 10, this may cause errors in the order if it's alphabetical.

Reply:
thanks it's indeed alphabetical